### ISSUE TITLE
Match shows error when running on new clean keychain

### ISSUE THREAD

**Issue by Antol** [fastlane/fastlane#5684]
### New Issue Checklist
- [x] Updated fastlane to the latest version
- [x] I have read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
##### output

```
[16:20:53]: --------------------------------
[16:20:53]: --- Step: create_keychain ---
[16:20:53]: --------------------------------
[16:20:53]: $ security list-keychains -d user
[16:20:53]: ▸ "/Users/antol/Library/Keychains/login.keychain"
[16:20:53]: -----------------------------
[16:20:53]: --- Step: match_all_certs ---
[16:20:53]: -----------------------------
[16:20:53]: $ security default-keychain
[16:20:53]: ▸ "/Users/antol/Library/Keychains/TempKeychain.keychain"
[16:20:53]: -------------------
[16:20:53]: --- Step: match ---
[16:20:53]: -------------------

+-----------------------+------------------------------------------------+
|                        Summary for match 0.6.2                         |
+-----------------------+------------------------------------------------+
| git_branch            | *********                                     |
| team_id               | **********                                     |
| app_identifier        | com.****.****.staging                     |
| type                  | enterprise                                     |
| readonly              | true                                           |
| shallow_clone         | true                                           |
| force_for_new_devices | false                                          |
| git_url               | git@bitbucket.org:*******/******-certs.git |
| username              | antol@******.com                       |
| keychain_name         | TempKeychain.keychain                          |
| verbose               | false                                          |
| force                 | false                                          |
| skip_docs             | false                                          |
+-----------------------+------------------------------------------------+

[16:20:53]: Cloning remote git repo...
[16:20:56]: Checking out branch *********...
[16:20:56]: 🔓  Successfully decrypted certificates repo
[16:20:56]: Installing certificate...

//Error
[16:20:56]: There are no local code signing identities found. You can run `security find-identity -v -p codesigning` to get this output. 
            This Stack Overflow thread has more information: http://stackoverflow.com/q/35390072/774. 
            (Check in Keychain Access for an expired WWDR certificate: http://stackoverflow.com/a/35409835/774 has more info.)

//Following output
[16:20:57]: Installing provisioning profile...

+----------------------+--------------------------------------------+
|                  Installed Provisioning Profile                   |
+----------------------+--------------------------------------------+
| App Identifier       | com.******.********.staging                 |
| Type                 | enterprise                                 |
| UUID                 | ********************************       |
| Environment Variable | sigh_com.*******.*******.staging_enterprise |
+----------------------+--------------------------------------------+

[16:20:57]: All required keys, certificates and provisioning profiles are installed 🙌
[16:20:57]: 🔒  Successfully encrypted certificates repo
[16:20:57]: Setting Provisioning Profile type to 'enterprise'
```
### Configuration Files

Fastfile:

``` ruby
lane :temp do
  create_keychain
  match
  delete_keychain
end
```

.env

``` ruby
MATCH_KEYCHAIN_NAME="TempKeychain.keychain"
KEYCHAIN_NAME="TempKeychain.keychain"
KEYCHAIN_PASSWORD="*******"
KEYCHAIN_SET_DEFAULT="1"
```
### Environment

/Users/antol/.rbenv/versions/2.2.4/lib/ruby/gems/2.2.0/gems/fastlane-1.99.0

Do you use bundler to execute fastlane (i.e. `bundle exec fastlane`)? 
- Yes

**Comment by yakimant** [comment]
I have the same error, when there are no codesigning identities in keychain.

**Comment by TKBurner** [comment]
@Antol @yakimant Have you run `match development` yet for this app? Also can you set `read_only: false`

Let me know how it goes! Hope you Have a great day!

**Comment by darrarski** [comment]
I had the same error. As a workaround, I created new development certificate manually using Apple Developer Portal, downloaded it and installed on my Mac. Then, I just run `match development` and `match appstore`. This solved the issue. After match downloaded my certificates stored in GitHub repository, I revoked newly created development certificate and removed it from my Mac.

I had one more problem described here: #5649, but creating symlink:
```
ln -s ~/Library/Keychains/login.keychain-db ~/Library/Keychains/login.keychain
```
solved it as well.

**Comment by TKBurner** [comment]
@darrarski Thanks for sharing that update. Are you using Sierra? Thanks!

**Comment by darrarski** [comment]
@TKBurner yes, I am using Sierra (clean install from usb stick after formatting whole drive). Apart from this issue I had no problems so far.

**Comment by TKBurner** [comment]
Hi @darrarski Thanks for following up. The symlink is required on sierra right now because of the change in location of the keychain. Do you still have any outstanding issues or are you able to run your lanes now? Thanks!

**Comment by darrarski** [comment]
@TKBurner you are right about the symlink. I have no problems running lanes on Sierra after I performed steps mentioned above in my comment.

**Comment by TKBurner** [comment]
@darrarski Thanks! Going to leave this open because of the outstanding issue with the keychain 👍

**Comment by KrauseFx** [comment]
This should have been fixed with the latest _fastlane_ update, let us know if you still experience any issues.

**Comment by i-schuetz** [comment]
I'm getting this error with CircleCI. Circle uses "circle" instead of login keychain, could this be related?

**Comment by TKBurner** [comment]
@i-schuetz Have you passed a keychain_name? Thanks!

**Comment by philipkaare** [comment]
I am experiencing a similar issue while setting up match on a clean Sierra machine with Fastlane 2.2.0 and Match 2.2.0

I am trying to set it up on an existing account, but ran nuke on both development and distribution to start on a clean slate. I have installed the WWDR certificate, both to the login and System keychains. 

I have also tried creating a development certificate as suggested above, but this also did not help, as there are no certificates in the git store - I am trying to make them. 

I experience the problem both when running `sudo fastlane match development` and `sudo fastlane match distribution`. 

Any help would be greatly appreciated, I spent most of a day struggling with this now.
